.ci/scripts/test.sh

```diff
@@ -250,6 +250,9 @@ docker-compose \
   --abort-on-container-exit \
   node_tests
 
+echo "Workaround issues with docker-compose down while removing network"
+docker network prune --force || true
```

Comment:
Does this actually force removal of the `docker_default` network? 

```
% docker network prune --help

Usage:  docker network prune [OPTIONS]

Remove all unused networks

Options:
      --filter filter   Provide filter values (e.g. 'until=<timestamp>')
  -f, --force           Do not prompt for confirmation
```

That command says it removes *unused* networks. Before `docker-compose down` is called, if I understand correctly, the `docker_default` network will still be in used.

Reply:
A possible alternative here would be something based on this comment: https://github.com/elastic/apm-agent-nodejs/pull/1974#issuecomment-830275247

For example, something like this:

```diff
diff --git a/.ci/scripts/test.sh b/.ci/scripts/test.sh
index 26af1d4e..9d39867e 100755
--- a/.ci/scripts/test.sh
+++ b/.ci/scripts/test.sh
@@ -250,8 +250,15 @@ docker-compose \
   --abort-on-container-exit \
   node_tests

-NODE_VERSION=${NODE_VERSION} docker-compose \
-  --no-ansi \
-  --log-level ERROR \
-  -f .ci/docker/${DOCKER_COMPOSE_FILE} \
-  down -v --remove-orphans
+if ! NODE_VERSION=${NODE_VERSION} docker-compose \
+    --no-ansi \
+    --log-level ERROR \
+    -f ${DOCKER_FOLDER}/${DOCKER_COMPOSE_FILE} \
+    down -v; then
+  # Workaround for this commonly seen error:
+  #   error while removing network: network docker_default id $id has active endpoints
+  echo "Unexpected error in 'docker-compose down ...'. Forcing removal of unused networks."
+  docker network inspect docker_default || true
+  docker network inspect -f '{{range .Containers}}{{ .Name }} {{end}}' docker_default || true
+  docker network prune --force || true
+fi
```

Thoughts?

Reply:
This seems like it is likely to work and there's certainly no harm in giving it a try. 👍

Reply:
Attempting this here: https://github.com/elastic/apm-agent-nodejs/pull/2184